go-controller/pkg/cni/helper_linux.go

```diff
@@ -381,10 +381,13 @@ func ConfigureOVS(ctx context.Context, namespace, podName, hostIfaceName string,
 		fmt.Sprintf("external_ids:attached_mac=%s", ifInfo.MAC),
 		fmt.Sprintf("external_ids:iface-id=%s", ifaceID),
 		fmt.Sprintf("external_ids:iface-id-ver=%s", initialPodUID),
-		fmt.Sprintf("external_ids:ip_addresses=%s", strings.Join(ipStrs, ",")),
 		fmt.Sprintf("external_ids:sandbox=%s", sandboxID),
 	}
 
+	if len(ifInfo.IPs) > 0 {
```

Comment:
nit: I would add a comment here. After years and years of no questions asked about IPAM, people might be surprised to see this.